src/genie/libs/parser/iosxe/show_ipv6_protocols.py

```diff
@@ -0,0 +1,979 @@
+''' show_ipv6_protocols.py
+
+IOSXE parsers for the following show commands:
+    * show ipv6 protocols
+'''
+
+# Python
+import re
+import xmltodict
+from netaddr import IPAddress
+
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Schema, Any, Or, Optional
+from genie.libs.parser.utils.common import Common
+
+
+# ==============================
+# Schema for 'show ipv6 protocols'
+# ==============================
+class ShowIpv6ProtocolsSchema(MetaParser):
+
+    ''' Schema for "show ip protocols" '''
+
+    schema = {
+        'protocols': {
+            Optional('rip'): {
+                'vrf': {
+                    Any(): {
+                        'address_family': {
+                            Any(): {
+                                Optional('instance'): {
+                                    Any(): {
+                                        Optional('output_delay'): int,
+                                        Optional('automatic_network_summarization_in_effect'): bool,
```

Comment:
I think this can also be removed. Correct me if I'm mistaken

Reply:
Please check to see if any other extraneous keys can be removed from this schema